.github/workflows/build-rpm.yml

```diff
@@ -34,6 +34,7 @@ jobs:
         run: mkdir -p ~/.config/
 
       - name: generate copr config file
+        if: github.ref == 'refs/heads/master'
```

Comment:
Is this the master branch of the kubo repository or the `redhat-et/ipfs-rpm` repo? `ipfs-rpm` uses `main` instead of `master`.